bucket/jmeter-pm.json

```diff
@@ -0,0 +1,37 @@
+{
+    "description": "The idea of JMeter Plugins Manager is simple: instead of installing various plugins manually, it will do it for you through nice UI. No need to mess with JARs anymore.",
+    "homepage": "https://jmeter-plugins.org/",
+    "version": "1.3",
+    "url": "http://search.maven.org/remotecontent?filepath=kg/apc/jmeter-plugins-manager/1.3/jmeter-plugins-manager-1.3.jar",
+    "hash": "482b8a9cd82a69ef1b633fc4235bfa9bee96894f663e5085df7061410d6ab99a",
+    "depends": "jmeter",
+    "installer": {
+        "script": [
+            "$jmeter_dir = resolve-path (split-path (split-path (scoop which jmeter)))",
```

Comment:
use [appdir](https://github.com/lukesampson/scoop/blob/ef97f0875ae39505adbae376c21f726484c1c65e/lib/core.ps1#L114) function

https://github.com/lukesampson/scoop-extras/search?q=appdir&unscoped_q=appdir
https://github.com/lukesampson/scoop/search?l=JSON&q=appdir

Reply:
Nice new feature, used in 10bf93b, thanks!